README.md

```diff
@@ -20,17 +20,25 @@ or for use without any build tools:
 
 ```html
 <script type="module">
-  import { webln } from "https://esm.sh/@getalby/sdk@3.5.1"; // jsdelivr.net, skypack.dev also work
+  import { webln } from "https://esm.sh/@getalby/sdk@3.6.0"; // jsdelivr.net, skypack.dev also work
```

Comment:
nwc needs to be imported, only webln is imported. And I think we should split this into separate examples